code/controllers/subsystem/SSair.dm

```diff
@@ -16,87 +15,65 @@ SUBSYSTEM_DEF(air)
 	runlevels = RUNLEVEL_GAME | RUNLEVEL_POSTGAME
 	offline_implications = "Turfs will no longer process atmos, and all atmospheric machines (including cryotubes) will no longer function. Shuttle call recommended."
 	cpu_display = SS_CPUDISPLAY_HIGH
-	var/cost_turfs = 0
-	var/cost_groups = 0
-	var/cost_highpressure = 0
+	var/cost_milla_tick = 0
```

Comment:
Can you autodoc these new variables, and/or the rest of the variablse here, if you understand what they're trying to do? Would probably make understanding what's going on in the subsystem a fair bit clearer